docs/docs/en/contribute/join/pull-request.md

```diff
@@ -64,7 +64,7 @@ Please refer to the commit message section.
 ### Pull Request Code Style
 
 DolphinScheduler uses `Spotless` to automatically fix code style and formatting errors,
-see [Code Style](../development-environment-setup.md#code-style) for details.
+see [Code Style](../development-environment-setup.md#Code Style) for details.
```

Comment:
It worked before. I'm not sure why it is not working. Pretty wired. But I don't think it is caused by `-`, I suppose the reason is `.md` cannot get mapped to `.html`. Could u plz help take a look when available? @zhongjiajie

Reply:
> annot ge

yes, i think so, becasue zh md is working fine.

Reply:
@DarkAssassinator @EricGao888 I have half remember that we do no support archor in our website https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/contribute/join/pull-request.html and it work in github source code page https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md

this is because we website template do not support it

Reply:
@zhongjiajie Thanks for the confirmation. I will submit a PR later to fix all the anchors. @DarkAssassinator Thanks again for reporting this issue : )